site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/model/SiteNameDTO.java

```diff
@@ -0,0 +1,20 @@
+package uk.ac.ox.ndph.mts.site_service.model;
```

Comment:
new siteName object that will only contain the site name and site id - the UI will display the name and possibly use the id as a link ion future development.